README.md

```diff
@@ -35,6 +35,26 @@ Windows:
 this should be the same as Linux & macos just with windows adjusted paths
 ```
 
+## Configuring IntelliJ
+
+You may prefer to run the backend using the IntelliJ IDE:
+
+1. Install the Scala plugin: file -> settings -> plugins
+2. Install Scala 2.13.10
+3. Set project SDK version to 11: file -> project structure -> select an SDK -> require version 11
+4. Enable 'Use plugin registry': file -> settings -> build, execution, deployment -> Maven
+5. Open Maven tab on the right and click install on vuu-parent -> lifecycle -> install
+6. In the terminal, navigate to 
+```sh
+vuu-ui/sample-apps/app-vuu-example
+```
+7. Run 
+```sh
+yarn build
```

Comment:
this is the same as the
```
yarn:build app
```
line in the client instructions below. This isn't wrong, of course, but won't work unless the user has done an initial 
```
yarn
```
to install everything. Probably best to just instruct user to follow the client installation below before running the vuu server for first time,.

Reply:
I've added the instruction and moved it below client installation so it makes chronological sense